sybil/integration/pytest.py

```diff
@@ -42,7 +42,7 @@ def toterminal(self, tw):
 
 class SybilItem(pytest.Item):
 
-    def __init__(self, parent, sybil, example) -> None:
+    def __init__(self, parent, sybil, example: example.Example) -> None:
```

Comment:
I added the type hint for `example` here so that the type checker knows what type `self.example.path` is below.

Reply:
```suggestion
    def __init__(self, parent, sybil, example: Example) -> None:
```

...with the requisite import change please.